changelog/fix-TEC-5401-add-redirects-for-all-tec-menu-options

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Added one time redirect to the First Time Setup page. [TEC-5401]
```

Comment:
Based on a convo today, there should be a trailing empty line at the end. [link](https://lw.slack.com/archives/C01H7Q57P1C/p1741793702942659)

Reply:
Github hides it here - look for the "no new line" symbol. If you use the changelogger the line is there, but won't show here.